src/lib/cares_reply.c

```diff
@@ -0,0 +1,534 @@
+/* Copyright (C) 2021 by Kyle Evans
+ *
+ * Permission to use, copy, modify, and distribute this software and its
+ * documentation for any purpose and without fee is hereby granted, provided
+ * that the above copyright notice appear in all copies and that both that
+ * copyright notice and this permission notice appear in supporting
+ * documentation, and that the name of M.I.T. not be used in advertising or
+ * publicity pertaining to distribution of the software without specific,
+ * written prior permission.  M.I.T. makes no representations about the
+ * suitability of this software for any purpose.  It is provided "as is"
+ * without express or implied warranty.
+ */
+
+#include "ares_setup.h"
+#include "ares.h"
+#include "ares_private.h"
+#include "string.h"
+
+
+cares_srv_reply*
+cares_srv_reply_get_next(const cares_srv_reply* srv_reply)
+{
+  return srv_reply->next;
+}
+
+const char* cares_srv_reply_get_host(const cares_srv_reply* srv_reply)
+{
+  return srv_reply->host;
+}
```

Comment:
Should we check for null on the input variables in these getters? Like on srv_reply here? It will be UB if a caller passes a null pointer, right?